climateset/__init__.py

```diff
@@ -2,6 +2,10 @@
 
 APP_ROOT = Path(__file__).resolve().parent
 PROJECT_ROOT = APP_ROOT.parent
-CONFIG = PROJECT_ROOT / "config"
+CONFIGS = PROJECT_ROOT / "configs"
 DATA_DIR = PROJECT_ROOT / "data"
+RAW_DATA = DATA_DIR / "raw"
+PROCESSED_DATA = DATA_DIR / "processed"
+LOAD_DATA = DATA_DIR / "load"
+META_DATA = DATA_DIR / "meta"
```

Comment:
Quick remark : there used to be a file called `data_paths.py` I just ported the important paths here instead with the other Path constants I already had